lib/spack/spack/test/cmd/commands.py

```diff
@@ -284,4 +284,18 @@ def test_updated_completion_scripts(shell, tmpdir):
 
     commands("--aliases", "--format", shell, "--header", header, "--update", new_script)
 
-    assert filecmp.cmp(old_script, new_script), msg
+    # TODO: changes in this PR add a site-admin config scope. If one runs
```

Comment:
```suggestion
    # TODO: changes in #47615 add a site-admin config scope. If one runs
```
Not sure this needs to get resolved in this PR. Just documenting history.